docs/concepts/loader.md

```diff
@@ -4,4 +4,22 @@ sidebar_position: 1
 
 # Loaders
 
-TODO
\ No newline at end of file
+The `loader` option is used to configure loaders, default being `I18nJsonLoader`. Currently `I18nJsonLoader` and `I18nYamlLoader` are supported out of the box.
+
+```typescript title="src/app.module.ts"
+       I18nModule.forRoot({
+          fallbackLanguage: 'en',
+          loaderOptions: {
+            path: path.join(__dirname, '/i18n/'),
+          },
+          loader: I18nYamlLoader,
+        }),
+```
+
+:::tip
+
+If you need to implement custom loaders , your resolver needs to extend  `src/loaders/i18n.abstract.loader.ts` interface. Also you can check the `src/loaders` to see how to default resolvers are implemented.
```

Comment:
This should say `your loader` instead of `your resolver`. Also change `default resolvers` to `default loaders`